scvi/module/_classifier.py

```diff
@@ -67,3 +67,14 @@ def __init__(
     def forward(self, x):
         """Forward computation."""
         return self.classifier(x)
+
+
+class LinearClassifier(nn.Module):
+    """Basic linear classifier."""
+
+    def __init__(self, n_input: int, n_labels: int):
+        super().__init__()
+        self.module = nn.Sequential(nn.Linear(n_input, n_labels), nn.Softmax(dim=-1))
+
+    def forward(self, x):
+        return self.module(x)
```

Comment:
can we just made n hidden layers = 0  in the default classifier be a linear classifier?

Reply:
the classifier code would need to change slightly for this to work

Reply:
I think it's worth it just to make the overall api simpler. Also the logits option is important to do things like logsoftmax for numerical stability